addons/mail/static/tests/composer/composer.test.js

```diff
@@ -1054,3 +1054,76 @@ test("Canned response last used changes on posting", async () => {
 });
+
+test("Canned response suggestions are not focused by default", async () => {
+    const pyEnv = await startServer();
+    const channelId = pyEnv["discuss.channel"].create({ name: "General" });
+    pyEnv["mail.canned.response"].create({ source: "Hello", substitution: "Hello! How are you?" });
+    await start();
+    await openDiscuss(channelId);
+    await insertText(".o-mail-Composer-input", "@");
+    await contains(".o-mail-NavigableList-item .o-mail-NavigableList-active", {
+        text: "Mitchell Admin",
+    });
+    await insertText(".o-mail-Composer-input", ":", { replace: true });
+    await contains(".o-mail-NavigableList-item .o-mail-NavigableList-active", {
+        text: "HelloHello! How are you?",
+        count: 0,
+    });
+});
+
+test("Focus first canned response suggestion when pressing TAB/ARROW keys", async () => {
+    const pyEnv = await startServer();
+    const channelId = pyEnv["discuss.channel"].create({ name: "General" });
+    pyEnv["mail.canned.response"].create([
+        { source: "Hello", substitution: "Hello! How are you?" },
+        { source: "Howdy", substitution: "Howdy! How are you?" },
+    ]);
+    await start();
+    await openDiscuss(channelId);
+    await insertText(".o-mail-Composer-input", ":");
+    await contains(".o-mail-NavigableList-item", {
+        text: "HelloHello! How are you?",
+        before: [".o-mail-NavigableList-item", { text: "HowdyHowdy! How are you?" }],
+    });
+    await contains(".o-mail-NavigableList-item .o-mail-NavigableList-active", { count: 0 });
+    await triggerHotkey("Tab");
+    await contains(".o-mail-NavigableList-item .o-mail-NavigableList-active", {
+        text: "HelloHello! How are you?",
+    });
+    await triggerHotkey("Escape");
+    await contains(".o-mail-NavigableList-item .o-mail-NavigableList-active", { count: 0 });
+    await insertText(".o-mail-Composer-input", ":", { replace: true });
+    await contains(".o-mail-NavigableList-item", {
+        text: "HelloHello! How are you?",
+        before: [".o-mail-NavigableList-item", { text: "HowdyHowdy! How are you?" }],
+    });
+    await triggerHotkey("ArrowDown");
+    await contains(".o-mail-NavigableList-item .o-mail-NavigableList-active", {
+        text: "HelloHello! How are you?",
+    });
+    await triggerHotkey("Escape");
+    await contains(".o-mail-NavigableList-item .o-mail-NavigableList-active", { count: 0 });
+    await insertText(".o-mail-Composer-input", ":", { replace: true });
+    await contains(".o-mail-NavigableList-item", {
+        text: "HelloHello! How are you?",
+        before: [".o-mail-NavigableList-item", { text: "HowdyHowdy! How are you?" }],
+    });
+    await triggerHotkey("ArrowUp");
+    await contains(".o-mail-NavigableList-item .o-mail-NavigableList-active", {
+        text: "HelloHello! How are you?",
+    });
+});
+
+test("Hide canned response suggestion when pressing enter without a selection", async () => {
```

Comment:
Would just name it `ENTER closes canned response suggestions`, brief and very clear